open-backport-pull-request/src/run.ts

```diff
@@ -55,34 +72,57 @@ const hasDiff = async (params: hasDiffParams): Promise<boolean> => {
 
 type openPullRequestParams = {
   headBranch: string
+  headSha: string
   baseBranch: string
   context: Context
   octokit: Octokit
 }
+
 const openPullRequest = async (params: openPullRequestParams): Promise<string | undefined> => {
+  // Create a working branch so that we can edit it if conflicted.
+  // Generally, the head branch is protected and cannot be edited.
+  const workingBranch = `backport-${params.headBranch.replaceAll('/', '-')}-${Date.now()}`
+  await params.octokit.rest.git.createRef({
+    owner: params.context.repo.owner,
+    repo: params.context.repo.repo,
+    ref: `refs/heads/${workingBranch}`,
+    sha: params.headSha,
+  })
+
+  core.info(`Creating a pull request ${workingBranch} -> ${params.baseBranch}`)
+  const pr = await params.octokit.rest.pulls.create({
+    owner: params.context.repo.owner,
+    repo: params.context.repo.repo,
+    head: workingBranch,
+    base: params.baseBranch,
+    title: `Backport ${params.headBranch} into ${params.baseBranch}`,
+    body: `Created from https://github.com/${context.repo.owner}/${context.repo.repo}/actions/runs/${context.runId}`,
+  })
+  core.info(`Created ${pr.data.html_url}`)
+
+  core.info(`Requesting a review to ${params.context.actor}`)
   try {
-    const pr = await params.octokit.rest.pulls.create({
+    await params.octokit.rest.pulls.requestReviewers({
       owner: params.context.repo.owner,
       repo: params.context.repo.repo,
-      head: params.headBranch,
-      base: params.baseBranch,
-      title: `Backport ${params.headBranch} into ${params.baseBranch}`,
-      body: `Created from https://github.com/${context.repo.owner}/${context.repo.repo}/actions/runs/${context.runId}`,
+      pull_number: pr.data.number,
+      reviewers: [params.context.actor],
     })
-    return pr.data.html_url
-  } catch (err) {
-    console.error(err)
+  } catch (e) {
+    core.info(`could not request a review to ${params.context.actor}: ${String(e)}`)
+  }
 
-    if (isUnprocessableEntityError(err)) {
-      core.warning('Pull Request already exists')
```

Comment:
@int128 
Why did you remove this error handling?

Reply:
Ah, the answer was here:
https://github.com/quipper/monorepo-deploy-actions/pull/745/files#r1002880273